test/storage/bwtree_key_test.cpp

```diff
@@ -221,13 +221,13 @@ class BwTreeKeyTests : public TerrierTest {
     transaction::TransactionManager txn_manager(&buffer_pool, true, LOGGING_DISABLED);
     storage::GarbageCollector gc_manager(&txn_manager);
 
+    auto *const txn = txn_manager.BeginTransaction();
+
     // dummy tuple to insert for each key. We just need the visible TupleSlot
-    auto *const insert_buffer = common::AllocationUtil::AllocateAligned(tuple_initializer.ProjectedRowSize());
-    auto *const insert_tuple = tuple_initializer.InitializeRow(insert_buffer);
+    auto *const insert_redo = txn->StageWrite(catalog::db_oid_t(0), catalog::table_oid_t(0), tuple_initializer);
```

Comment:
Same comment as above about using a `#define` clause for tests and benchmarks so that it is easier to reason about how the OIDs compare to what the catalog will eventually assign.

Reply:
Where do we want it defined? Which util?

Reply:
`catalog_[test|benchmark]_util.h`?  It would be a new file on the benchmark side, but I think that (or something like it) would be a good place for putting catalog mock-ups and helpers in the future.